pkg/common/errors.go

```diff
@@ -24,4 +24,9 @@ import "errors"
 var (
 	// InvalidQueueName returned when queue name is invalid
 	InvalidQueueName = errors.New("invalid queue name, max 64 characters consisting of alphanumeric characters and '-', '_', '#', '@', '/', ':' allowed")
+
+	PreemptionPreconditionsFailed = errors.New("PreemptionPreconditionsFailed")
+	PreemptionDoesNotGuarantee    = errors.New("PreemptionQueueGuaranteesCheckFailed")
+	PreemptionShortfall           = errors.New("PreemptionHelpedButShortfallOfResources")
+	PreemptionDoesNotHelp         = errors.New("PreemptionDoesNotHelp")
```

Comment:
Just use string constants

Reply:
Actually I just realized that it's better if the string literals are complete sentences like "Preemption Preconditions Failed", "Preemption queue guarantees check failed", etc because that will be exposed to YK users.